app/views/practices/form/about.html.erb

```diff
@@ -63,10 +63,13 @@
                       <li class="practice-editor-about-li fields" <%= "id=practice_va_employees_attributes_#{vae.index}_li" if vae_id %> role="option" data-id="<%= vae_id %>">
                         <div class="grid-row">
                           <div class="grid-col-12">
-                            <%= vae.label :name, 'Name', class: 'usa-label x0-top' %>
+                            <div title="Type the name of the team member.">
```

Comment:
I just made a commit with these changes..  i..e, Place helper text below label in about team members section.  In the practice Show view - Core resources... check if attachment or link exist instead of name when displaying this section. If name or description is blank fill with either 'attachment_file_name' or 'link_url'